exporters/metric/datadog/datadog.go

```diff
@@ -0,0 +1,178 @@
+package datadog
```

Comment:
@lizthegrey can you comment on license and copyright requirements for packages in this repo?

Reply:
Vendor code can keep its past licensing statement module-by-module, although we'd prefer it have future ongoing authorship attributed to the OpenTelemetry Contributors.

Reply:
Personally, I keep Honeycomb's exporter in its own repo so we don't have to wait for releases of -contrib.

Reply:
Thank you both, just let me know if I need to add/change anything to this file or move the package to another repo 🙂 happy to do w/e!

Reply:
DataDog is not here and hasn't responded in the past. I'm comfortable putting this here until DataDog would like us to remove it in favor of a repository of theirs.